functions/__fzf_preview_file.fish

```diff
@@ -1,17 +1,26 @@
 # helper function for __fzf_search_current_dir
 function __fzf_preview_file --description "Prints a preview for the given file based on its file type." --argument-names file_path
-    set file_type (file --brief -i "$file_path")
+    if test -d "$file_path"
+        set file_type "d"
+    else
+        set file_type (ls -o "$file_path" | cut -c 1) #exa is faster than ls and file
```

Comment:
So a clean installation of exa aliases itself to ls?

Reply:
No, I am using exa on my machine and you would have to install exa, its a single binary. You would have to either:
+ replace all the instance of ls manually in the file
+ idk if aliases would work, i use global abbr, so ls is replaced in shell with exa but idk how to make it work with scripts. 

AFAIK, ls was 10x slower than file, but since file was running in micro seconds per file, that makes ls still in milli seconds.